doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -99,14 +99,26 @@ Keywords
   The following character sequences are reserved keywords that cannot be
   used as identifiers::
 
-    _ Axiom CoFixpoint Definition Fixpoint Hypothesis IF Parameter Prop
-    SProp Set Theorem Type Variable as at by cofix discriminated else
-    end exists exists2 fix for forall fun if in lazymatch let match
-    multimatch return then using where with
+    _ Axiom CoFixpoint Definition Fixpoint Hypothesis Parameter Prop
+    SProp Set Theorem Type Variable as at cofix else
+    end fix for forall fun if in let match
+    return then where with
+
+  The following identifiers are not reserved keywords but are
+  part of the definition of Gallina::
+
+    Admitted Axiom Axioms Class CoFixpoint CoInductive Conjecture
+    Conjectures Constraints Corollary Cumulative Defined Definition
+    Example Fact Fixpoint Global Hypotheses Hypothesis Inductive Lemma
+    Let Local Monomorphic NonCumulative Parameter Parameters
+    Polymorphic Private Proof Property Proposition Record Register Qed
+    Remark Structure SubClass Theorem Universe Universes Variable
+    Variables Variant
+
+  They should not be used as keywords in :cmd:`Notation`.
```

Comment:
```suggestion

```

(moved above)

Reply:
This is an old comment. I don't think we need to forbid them in `Notation` after all (see @SkySkimmer's comment).